<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HyperMark</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QFrame" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>751</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>791</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>771</width>
       <height>511</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="widget_6" native="true">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>40</y>
           <width>181</width>
           <height>80</height>
          </rect>
         </property>
         <widget class="QPushButton" name="templateBtn">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>131</width>
            <height>41</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>TEMPLATE</string>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="text2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>541</width>
           <height>141</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Select your HTML template file and insert &quot;{content}&quot; where you want to inject the Markdown content in your HTML template file.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_8">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="widget_7" native="true">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>40</y>
           <width>181</width>
           <height>80</height>
          </rect>
         </property>
         <widget class="QPushButton" name="sourceBtn">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>131</width>
            <height>41</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>SOURCE</string>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="text3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>541</width>
           <height>141</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Select the directory containing the Markdown files to be converted, and convert all Markdown files in that directory, including those in different subdirectories. After conversion, an HTML directory will be automatically created in this directory to store the converted HTML files.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_14">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="widget_9" native="true">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>40</y>
           <width>181</width>
           <height>80</height>
          </rect>
         </property>
         <widget class="QPushButton" name="transformBtn">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>131</width>
            <height>41</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>TRANSFORM</string>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="text4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>541</width>
           <height>101</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>To start the conversion of Markdown files to HTML files.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images/logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>153</x>
       <y>41</y>
       <width>601</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images/word.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/close.png</normaloff>images/close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
